developer/src/kmc/src/commands/build.ts

```diff
@@ -60,9 +60,9 @@ If no input file is supplied, kmc will build the current folder.`)
   buildCommand
     .command('windows-package-installer <infile>')
     .description('Build an executable installer for Windows for a Keyman package')
-    .option('--msi <msiFilename>', 'Location of keymandesktop.msi')
-    .option('--exe <exeFilename>', 'Location of setup.exe')
-    .option('--license <licenseFilename>', 'Location of license.txt')
+    .requiredOption('--msi <msiFilename>', 'Location of keymandesktop.msi')
+    .requiredOption('--exe <exeFilename>', 'Location of setup.exe')
+    .requiredOption('--license <licenseFilename>', 'Location of license.txt')
```

Comment:
How specific are the filenames?

From the reported issue, where they using `setup-redist.exe` and `LICENSE.md`?

Reply:
The filenames are not important